app/scripts/views/form.js

```diff
@@ -199,6 +199,10 @@ define(function (require, exports, module) {
 
     _submitForm: notifyDelayedRequest(showButtonProgressIndicator(function () {
       var self = this;
+      // sets all password fields to type=password
+      self.$el.find('.password').each(function (i, el) {
+        el.type = self.getElementType('.password');
```

Comment:
Using `self.getElementType('.password')` here is a bit confusing. getElementType looks up a series of password elements. should this just use `'password'`?